packages/acceptit-Tests.package/ACReadMeFactory.class/class/addLibraryAndScenarioToUs.st

```diff
@@ -1,6 +1,6 @@
 as yet unclassified
 addLibraryAndScenarioToUs
-	userStory class
+	userStory theMetaClass
```

Comment:
what does the theMetaClass call do?

Reply:
it's basically the same as `class` but as you can see travis is still red (but tests run locally) the problem seems to be in this method so I am experimenting what it could be.